sktime/classification/tests/test_all_classifiers.py

```diff
@@ -104,8 +104,8 @@ def test_classifier_output(self, estimator_instance, scenario):
             X_train = scenario.args["fit"]["X"]
             train_proba = estimator_instance._get_train_probs(X_train, y_train)
 
-            assert isinstance(y_proba, np.ndarray)
-            assert y_proba.shape == (X_new_instances, n_classes)
+            assert isinstance(train_proba, np.ndarray)
```

Comment:
this was a bug, correct?

For robustness, kindly use
```python
        _, _, X_train_metadata = check_is_scitype(X_train, "Panel", return_metadata=True)
        X_train_len = X_train_metadata["n_instances"]
```

This should ensure the test is still correct in case the mtype of `X_train` changes

Reply:
Done